components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1525,12 +1525,16 @@ void RewardsServiceImpl::OnGetConfirmationsHistory(
     callback.Run(0, 0.0);
   }
 
-  int total_viewed = info->transactions.size();
   double estimated_earnings = 0.0;
-  if (total_viewed > 0) {
-    for (const auto& transaction : info->transactions) {
-      estimated_earnings += transaction.estimated_redemption_value;
+  int total_viewed = 0;
+
+  for (const auto& transaction : info->transactions) {
+    if (transaction.confirmation_type != "view") {
```

Comment:
can we move this string into variable, so that we have it only in one place?

Reply:
👍 fixed